server/controllers/sendgrid.js

```diff
@@ -0,0 +1,58 @@
+const asyncHandler = require("express-async-handler");
+const sgMail = require('@sendgrid/mail')
+
+const User = require("../models/User");
+const Contest = require("../models/Contest");
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY)
+
+exports.sendWinnerEmail =  asyncHandler(async (req, res)=> {
+    const userID = req.user.id;
+    const contestID = req.params.id;
+    let success = false;
+    const { receiverID } = req.body;
+    try {
+        const user = await User.findById(userID);
```

Comment:
What is the reason for searching up a 'user' here? If he doesn't exist, how would he have logged in? :)